### ISSUE TITLE
Window controls in macOS toggle their hover state all at once

### ISSUE THREAD

**Issue by LiquidPL** [Outsetini/Air-for-Steam#164]
When unhovered:

![image](https://user-images.githubusercontent.com/991831/31035505-8d176f7c-a568-11e7-8bb2-bc02bff58adf.png)

When hovered (any of the buttons):

![image](https://user-images.githubusercontent.com/991831/31035581-d11de25a-a568-11e7-92b7-ad4df6ee924f.png)

Also I think that the macOS version of the theme should use macOS-style window controls, as in the [three circular buttons](https://user-images.githubusercontent.com/991831/31035650-168d0e60-a569-11e7-88d4-60c7bf769673.png) (red, yellow, green for close, minimize, maximize respectively).

**Comment by Outsetini** [comment]
I'm not 100% sure (since I don't have a Mac), but I think the group hover is how Steam itself handles the states. Either way, Air was built with Windows in mind and Mac isn't fully supported at this point. If yourself, or anyone else, wants to contribute some time to making the macOS version more viable, please feel free to give it a go and drop in some pull requests!

**Comment by kevzdev** [comment]
macOS handles the on-hover effects for these "close-minimize-maximize" buttons all at once, as seen here:
![not hovering](https://i.imgur.com/R7h4OgW.png) ![hovering](https://i.imgur.com/WnefhQl.png).

Hovering one of the buttons will trigger all of the 3 button's on-hover effect. That part can't be changed.

**Comment by LiquidPL** [comment]
Yep, totally forgot about this. Guess we could keep this open for the purpose of implementing proper macOS-style window controls, then.
